README.md

```diff
@@ -70,7 +70,7 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
     service YourService {
    -  rpc Echo(StringMessage) returns (StringMessage) {}
    +  rpc Echo(StringMessage) returns (StringMessage) {
-   +    option (gengo.grpc.gateway.ApiMethodOptions.api_options) = {
+   +    option (google.api.http) = {
    +      path: "/v1/example/echo"
    +      method: "POST"
```

Comment:
Doesn't this part of the example need to be updated to match the Google api definitions?

If I understand correctly, it should now be:

```
post: "/v1/example/echo"
body: "*"
```